pegasus/sites.v3/code.org/public/tos.md

```diff
@@ -72,17 +72,17 @@ You hereby grant to Code.org a limited, non-exclusive, sublicensable (as necessa
+By uploading, creating, submitting or distributing User Content through the Services and consenting to their public display (e.g., in featured projects), you hereby grant other Users a non-exclusive, transferable (only to a successor), royalty-free right and license to use, reproduce, create derivative works, publicly display and perform any of your User Content that is incorporated into that User's project, provided that Code.org shall not use or distribute your User Content on a stand-alone basis if Code.org’s license grant has been terminated as set forth below. The foregoing license and restrictions does not limit Code.org rights under any third-party or open-source license in which your User Content may be licensed.
 
 ##### Public Content
 
-In addition to the rights, licenses and privileges referred to above, you agree that Code.org may use and refer to your User Content, first name and screenname that are posted publicly on the Code.org Website and not in private areas of our Service (“Public Content”) (including screenshots) in marketing materials, press releases, financial reports, presentations, website materials, customer lists and other media now known or hereafter discovered in connection with the marketing, advertising and promotion of the  Service, and any products, goods, features, capabilities and/or services associated with the Service. This right will continue until your Public Content is deleted; provided however, that any Public Content previously printed in brochures, articles or other marketing materials that Code.org had prepared before any request for deletion was received will continue to be used.
+In addition to the rights, licenses and privileges referred to above, you agree that Code.org may use and refer to your User Content, first name and screenname that are posted publicly on the Code.org Website and not in private areas of our Service (“Public Content”) (including screenshots) in marketing materials, press releases, financial reports, presentations, website materials, customer lists and other media now known or hereafter discovered in connection with the marketing, advertising and promotion of the Service, and any products, goods, features, capabilities and/or services associated with the Service. This right will continue until your Public Content is deleted; provided however, that any Public Content previously printed in brochures, articles or other marketing materials that Code.org had prepared before any request for deletion was received will continue to be used.
```

Comment:
Now I'm not sure what a 'screenname' is but I assume it's also a legal term

Reply:
I'll ask, good call!